packages/atlas/src/components/Tabs/Tabs.styles.ts

```diff
@@ -20,7 +20,7 @@ type TabsGroupProps = {
   }
 }
 
-const getMaskImage = ({ shadowsVisible }: TabsGroupProps) => {
+export const getMaskImage = ({ shadowsVisible }: TabsGroupProps) => {
```

Comment:
Let's move it to the helpers or utils if this is used in two different components.